splunk_add_on_ucc_framework/__init__.py

```diff
@@ -310,23 +313,29 @@ def _install_libs(requirements, ucc_target, installer="python3"):
     else:
         logging.info(f"  Not using common requirements")    
 
-    if os.path.exists(os.path.join(path,"lib","py2", "requirements.txt")):
-        logging.info(f"  Uses py2 requirements")    
-        _install_libs(requirements=os.path.join(path,"lib","py2", "requirements.txt"), installer="python2", ucc_target=os.path.join(ucc_lib_target, "py2"))
-    elif os.path.exists(os.path.join(os.path.abspath(os.path.join(path, os.pardir)), "requirements_py2.txt")):
-        logging.info(f"  Uses py2 requirements")    
-        _install_libs(requirements=os.path.join(os.path.abspath(os.path.join(path, os.pardir)), "requirements_py2.txt"), installer="python2", ucc_target=os.path.join(ucc_lib_target, "py2"))        
-    else:
-        logging.info(f"  Not using py2 requirements")    
-
-    if os.path.exists(os.path.join(path, "lib","py3","requirements.txt")):
-        logging.info(f"  Uses py3 requirements")            
-        _install_libs(requirements=os.path.join(path,"lib", "py3","requirements.txt"), ucc_target=os.path.join(ucc_lib_target, "py3"))
-    elif os.path.exists(os.path.join(os.path.abspath(os.path.join(path, os.pardir)), "requirements_py3.txt")):
-        logging.info(f"  Uses py3 requirements")    
-        _install_libs(requirements=os.path.join(os.path.abspath(os.path.join(path, os.pardir)), "requirements_py3.txt"), installer="python3", ucc_target=os.path.join(ucc_lib_target, "py2"))        
-    else:
-        logging.info(f"  Not using py3 requirements")    
+
+    #Prevent certain packages from being included pip could be dangerous others are just wasted space
+    noshipdirs = ['setuptools', 'bin', 'pip', 'distribute', 'wheel']
+    p = Path(ucc_lib_target)
+    try:
+        for nsd in noshipdirs:
```

Comment:
Do we need to swap `try` and `for`?

It does not work if you have `lib/requirements.txt` with the following content:
```
setuptools
pip
wheel
```
In the resulting package we will still have `pkg_resources/`, `_distutils_hack/` folders and `distutils-precedence.pth` file.

Reply:
We may be able to drop those also but I wasn't positive and didn't want to go to far at this point